src/channel.c

```diff
@@ -4914,6 +4926,22 @@ get_job_options(typval_T *tv, jobopt_T *opt, int supported, int supported2)
 		opt->jo_set2 |= JO2_TERM_KILL;
 		opt->jo_term_kill = tv_get_string_chk(item);
 	    }
+	    else if (STRCMP(hi->hi_key, "term_mode") == 0)
+	    {
+		char_u *p;
+
+		if (!(supported2 & JO2_TERM_MODE))
+		    break;
+		opt->jo_set2 |= JO2_TERM_MODE;
+		p = tv_get_string_chk(item);
+		if (p == NULL || !(*p == NUL || STRCMP(p, "winpty") == 0
+					          || STRCMP(p, "conpty") == 0))
```

Comment:
```
if (p != NULL && *p != NUL
	&& STRCMP(p, "winpty") != 0 && STRCMP(p, "conpty") != 0)
```

Reply:
Ah... I am sleeping. to correct.

Reply:
No, was it right? It is certain that it is hard to understand. I want to make it easy to see.

Reply:
Ah, sorry. I'd slept.